src/DataValidation.py

```diff
@@ -1,519 +1,480 @@
-import logging

```

Comment:
What's going on with the diff here?

Reply:
Oh, we went from Windows line endings to Linux ones (`\r\n` to `\n`)

Reply:
Idk why, since im on Windows...